src/csvcubed/models/sparqlresults.py

```diff
@@ -143,14 +115,21 @@ class CodelistsResult:
     Model to represent select codelists sparql query result.
     """
 
-    codelists: list[CodelistResult]
+    codelists: List[CodelistResult]
     num_codelists: int
 
     @property
     def output_str(self) -> str:
+        def alter_code_list_for_text_representation(code_list: CodelistResult) -> Dict:
```

Comment:
Since we're doing work here, can we get rid of the `output_str` property and move it into the MetadataPrinter (or other appropriate location)?